nemo_text_processing/text_normalization/de/taggers/electronic.py

```diff
@@ -72,7 +78,28 @@ def __init__(self, deterministic: bool = True):
         protocol = pynutil.insert('protocol: "') + protocol + pynutil.insert('"')
         url = protocol + insert_space + (domain_graph)
 
-        graph = url | domain_graph | email | tag
+        # Implements a graph for commonly-used hyphenated compound nouns (e.g. 3D-Drucker, 2D-Mammogram)
+        graph_abbreviation = pynini.string_file(get_abs_path("data/electronic/abbreviations.tsv"))
```

Comment:
Mmm, thoughts on doing a more comprehensive pattern? Technically we can extend to any use of `[0-9]+-[A-Za-z]` Can you see potential conflict with that?

Reply:
Also reverse, would allow capturing things like G-9 summit, COVID-19.

Reply:
Mmm, my vote is trying out but make sure it's a change that can be commented out. That way the fix can be quick if we break anything.